test/ex_webrtc/peer_connection_test.exs

```diff
@@ -36,57 +36,76 @@ defmodule ExWebRTC.PeerConnectionTest do
                  fingerprint: {:sha256, @fingerprint}
                )
 
-  test "negotiation needed" do
```

Comment:
I merged this test with other already existing tests (e.g/ add_track, remove_track) or created new ones (e.g add_transceiver)